CONTRIBUTING.md

```diff
@@ -1,16 +1,8 @@
-# Contributing
+# Contributing to Swift Evolution
 
-## Participating in the Swift Evolution Process
+Before you initiate a pull request, please read:
 
-See [Participating in the Swift Evolution Process](https://swift.org/contributing/#participating-in-the-swift-evolution-process) on Swift.org and [process.md](process.md).
+* [Swift Evolution Process](process.md)
+* [Commonly Rejected Changes](commonly_proposed.md)
```

Comment:
Since there are only two items in this list, you might consider:

```markdown
Before you initiate a pull request, please read [Swift Evolution Process](process.md) and [Commonly Rejected Changes](commonly_proposed.md).
```

Reply:
GitHub currently gives `CODE_OF_CONDUCT.md` documents special UI on the root project page, sort of like how it directs people to `CONTRIBUTING.md` for new PRs, so maybe that's a better destination for that link. 

I'm not sure either way on the Contributing page; it's broadly useful but there's not much specific to Swift Evolution there.